src/bitmap/mutable.rs

```diff
@@ -94,6 +94,21 @@ impl MutableBitmap {
         self.length += 1;
     }
 
+    /// Pop the last bit from the [`MutableBitmap`].
+    /// # Panics
+    /// Panics iff the length is zero.
```

Comment:
should we also make this to return `Option<bool>`, to be consistent with [`Vec::pop`](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.pop)?

Reply:
It is both ok to me.

Since `MutableBitmap` is semantically equivalent to [`Vec<bool>`] as the comment said, let's use `Option<bool>`.
https://github.com/jorgecarleitao/arrow2/blob/d29e135c00cb862f939d67e965b6bb167e550951/src/bitmap/mutable.rs#L10-L11